PubNubSwiftConsole/Classes/UI/CollectionView/Cells/SubscribeStatusCollectionViewCell.swift

```diff
@@ -86,6 +100,18 @@ class SubscribeStatusCollectionViewCell: CollectionViewCell {
         } else {
             timeTokenLabel.hidden = true
         }
+        if item.client.isSubscribingToChannels, let channelsLabelString = item.client.channelsString() {
+            channelLabel.hidden = false
```

Comment:
i think you misunderstood me, this should use the values from subscribeStatus.channels, not from the client